project/sites/pressclippingsnics/themes/pressclippingsnics_theme/src/scss/components.scss

```diff
@@ -20,3 +20,22 @@
 #headerHero {
   background: none;
 }
+
+#block-nicsloginnotice {
+  background: $grey-medium;
+  text-align: center;
+
+  h2 {
+    @include adjust-font-size-to(30px);
+  }
+
+  p {
+    @include adjust-font-size-to(16px);
+  }
+}
+#header {
+  background: #f8f8f8;
```

Comment:
Should maybe add these colours as variables?

Reply:
Done thanks Adam